plugins/executor.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
 
+const Annotations = require('../config/annotations');
 const Joi = require('joi');
 const models = require('../models');
 const buildId = Joi.reach(models.build.base, 'id').required();
 const SCHEMA_START = Joi.object().keys({
+    annotations: Annotations.annotations,
```

Comment:
When we get to the `start` function, isn't it already **inside** the executor plugin? I'm a little confused on how this would work. Could you give some details on how this is going to be used?